build.cmd

```diff
@@ -1,9 +1,39 @@
 @echo Off
+setlocal enableextensions enabledelayedexpansion
 
 set password=%1
 
 set deployUrl=%2
 if "%deployUrl%" == "" (
    set deployUrl=https://npe.codeplex.com/releases/clickonce/
 )
```

Comment:
While you're editing this file, can you pls remove the clickonce url? No longer needed as no more clickonce deployment